model/src/main/scala/org/broadinstitute/dsde/rawls/model/UserModel.scala

```diff
@@ -5,6 +5,15 @@ import spray.json._
 sealed trait UserAuthRef
 case class RawlsUserRef(userSubjectId: RawlsUserSubjectId) extends UserAuthRef
 case class RawlsGroupRef(groupName: RawlsGroupName) extends UserAuthRef
+case class RawlsRealmRef(realmName: RawlsGroupName) extends UserAuthRef
+
+object RawlsGroupRef {
+  implicit def toRealmRef(ref: RawlsGroupRef): RawlsRealmRef = RawlsRealmRef(ref.groupName)
```

Comment:
Some conversions that this method allows will be invalid: a non-Realm Group to a Realm.  I am concerned that this will cause hard-to-debug problems.  Is it necessary?